modules/planning/scenarios/lane_follow/lane_follow_stage.cc

```diff
@@ -287,8 +287,9 @@ void LaneFollowStage::PlanFallbackTrajectory(
   }
 
   AERROR << "Speed fallback due to algorithm failure";
+  // TODO(Jiacheng): move this stop_path_threshold to a gflag
   const double stop_path_distance =
-      reference_line_info->path_data().discretized_path().Length();
+      reference_line_info->path_data().discretized_path().Length() - 5.0;
```

Comment:
Name magic numbers.

Reply:
I will take care of this. I'm working on code cleaning and GFLAGS making. He added a TODO with my name so I won't forget about this.